.github/workflows/main.yml

```diff
@@ -93,7 +93,7 @@ jobs:
               abi: 'x86_64'
               tag: 'google_apis'
 
-            - api: '32'
+            - api: '33'
```

Comment:
you should add a new version to support, not replace one.

Reply:
It might be worth to add a comment explaining why it is skipped then.

Reply:
Added comment